BUILD.md

```diff
@@ -236,9 +246,9 @@ sudo dnf install gcovr lcov
 Run tests and generate report:
 
 ``` shell
-meson setup -Dbuildtype=debug -Db_coverage=true build
-meson test -C build
-meson compile -C build coverage-html
+meson setup -Dbuildtype=debug -Db_coverage=true build/debug
+meson test -C build/debug
+meson compile -C build/debug coverage-html
```

Comment:
This doesn't work @kcgen; could you perhaps update it to the correct invocation?

Reply:
Needs either the Clang `lcov` package (which includes both the lcov binary and GenHTML binary), or the GNU `gcovr` package, which can do html generation itself.

```shell
$ meson setup <other flags> -Db_coverage=true build/cov
$ meson compile -C build/cov
$ meson test -C build/cov
$ meson compile coverage-html (or coverage-xml) -C build/cov
```

https://mesonbuild.com/howtox.html#producing-a-coverage-report

Also testing on my side.

Reply:
Ok, working on my side.

Reply:
Cheers, adding.